pkg/chartvalues/apiextensions_release_e2e_test.go

```diff
@@ -8,12 +8,18 @@ import (
 
 func newAPIExtensionsReleaseConfigFromFilled(modifyFunc func(*APIExtensionsReleaseE2EConfig)) APIExtensionsReleaseE2EConfig {
```

Comment:
Just realized it now. The `FromFilled` naming is weird. I do not know what it means.

Reply:
I agree that it is weird, the naming was introduced by the aws chart. 
`FromFilled` refers to the fact that the `APIExtensionsReleaseE2EConfig` is filled with correct values at first and then individual values are overwritten to test specific error cases.
I couldn't think of a better name though, please give a suggestion if it's a problem for you.

Reply:
I created the name. The idea is as @MarcelMue said: creating value by modifying correctly filled one. W/o this function tests would be several times longer and more error prone.

Reply:
That is maybe personal but the name is still weird. I don't get what it wants to tell me. No blocker though now.